1_space_odyssey/js/Hani-9000/app.js

```diff
@@ -0,0 +1,183 @@
+  */
+
+/* ______   __  __   ______  ______
+        _..._
+      .'     '.      _
+     /    .-""-\   _/ \
+   .-|   /:.   |  |   |
+   |  \  |:.   /.-'-./
+   | .-'-;:__.'    =/
+   .'=  *=|Noroff_.='
+  /   _.  |    ;
+ ;-.-'|    \   |
+/   | \    _\  _\
+\__/'._;.  ==' ==\
+         \    \   |
+         /    /   /
+         /-._/-._/
+  ...    \   `\  \
+          `-._/._/
+ 1. CREATE DROID::::::
+    Below are all the variables needed to initialize an android into consciousness in its default state.
+    a. assign the correct values to the variables below,
+    b. ensure to choose the correct values and datatypes
+    c. the droid should be built in your likeness (a human, with human features, 2 arms, legs etc)
+*/
+
+var isAlive = true;
+var name = "Hani-9000";
+var hasJetPack = false;
+var suitColor = "grey";
+var eyes = 2;
+var eyeColor = "brown";
+var hairColor = "black";
+var legs = 2;
+var arms = 2;
+var hands = 2;
+var ears = 2;
+var canWalk = true;
+var canRun = true;
+var canFly = false;
+var hasWeapon = false;
+var isInGoodMood = false;
+
+console.log(".-.-. Booting .-.-.");
+console.log(name, isAlive, hasJetPack, eyes, eyeColor, hairColor, legs, arms, hands, ears, canWalk, canFly, hasWeapon);
+
+if(hasJetPack === true) {
+    console.log("The Jetpack is enabled, off you go");
+    } else {
+    console.log("the jetpack is disabled, please enable it before you go out");
+}
+/*
+ 2. BROKEN SATELLITE::::::
+    You need to send the droid out on a space walk to fix the satellite, with out it you can't communicate with earth.
+    a) write an if condition checking if the jet pack is enabled, if true the droid can go outside if false the droid
+       cannot go outside, console log the correct log based on that condition.
+*/
+console.log(
+    "\u{1F6F0} BROKEN SATELLITE::::::"
+  );
+  
+  /*
+   uncomment these console logs and write your if else statement and use them to console log out the correct statement
+  
+   console.log("JetPack is enabled!!! off I go outside to fix the satellite");
+   console.log("JetPack NOT enabled, please enable jetPack before doing a space walk");
+   */
+  
+  /*
+   3. METEOR SHOWER::::::
+      On your way to Jupiter you encounter a massive meteor shower that could damage the ship, you need to redirect the ship to a new safe location.
+      a) Using a for loop iterate through the responses array and choose the appropriate response to get out of danger.
+        create an if condition in your loop to print only 1 response using a comparison operator.
+  */
+  console.log(
+    "\u{1F327} METEOR SHOWER::::::"
+  );
+  
+  var responses = [
+    "Call basecamp and ask for assistance, but they are far away",
+    "Do nothing, just push through",
+    "initiate core thrusters, punch 180degrees into dash-nav and warp away from danger",
+    "ask humanoid-9000 to fly out there and shield the ship from the debris, this action could harm the humanoid"
+  ];
+  var response = responses.Length;

```

Comment:
nice job yeah man.
spelling error: `esponses.Length;`
should be: `var response = responses.length;`

![https://media.giphy.com/media/rvoMMtlqibZYY/giphy.gif](https://media.giphy.com/media/rvoMMtlqibZYY/giphy.gif)